wfe2/wfe_test.go

```diff
@@ -1746,8 +1754,50 @@ func TestAccount(t *testing.T) {
 	}`)
 }
 
+type mockSAWithCert struct {
+	core.StorageGetter
+	cert   *x509.Certificate
+	status core.OCSPStatus
+}
+
+func newMockSAWithCert(t *testing.T, sa core.StorageGetter, valid bool) *mockSAWithCert {
+	status := core.OCSPStatusGood
+	if valid == false {
+		status = core.OCSPStatusRevoked
+	}
+	cert, err := core.LoadCert("../test/hierarchy/ee-r3.cert.pem")
+	test.AssertNotError(t, err, "Failed to load test cert")
+	return &mockSAWithCert{sa, cert, status}
+}
+
+// GetCertificate returns a hard-coded cert (ee-r3.cert.pem) which was issued by
```

Comment:
Two part nit: 
1) hard-coding a filename into the comment means more updating if this changes
2) this function doesn't actually specify the filename, it returns a `core.Certificate` based on the hard-coded filename in the `mockSAWithCert` constructor `newMockSAWithCert`

Reply:
Good call, updated the comment.